src/internal_events/prometheus.rs

```diff
@@ -129,19 +131,13 @@ impl InternalEvent for PrometheusRemoteWriteSnapError {
 
 #[derive(Debug)]
 pub struct PrometheusRemoteWriteReceived {
-    pub byte_size: usize,
     pub count: usize,
 }
 
 impl InternalEvent for PrometheusRemoteWriteReceived {
     fn emit_logs(&self) {
         debug!(message = "Received remote_write events.", count = ?self.count);
     }
-
-    fn emit_metrics(&self) {
-        counter!("processed_events_total", self.count as u64);
-        counter!("processed_bytes_total", self.byte_size as u64);
-    }
```

Comment:
Why aren't metrics sent here?

Reply:
`prometheus_remote_write` source is using shared `HttpSource` which is emitting those metrics with `HTTPEventsReceived` so this `emit_metrics` was deleted to remove double counting.